primitives/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "moved-primitives"
```

Comment:
I'm not sure if primitives should be its own crate, but then again why not. @birchmd what do you think?

Reply:
I think this is done to prevent dependency cycles among our crates now that the big `moved` crate has been separated out (which was necessary to make the genesis image crate effective). So it makes sense to have primitives as separate crate now.